charmcraft/commands/store/__init__.py

```diff
@@ -285,3 +285,99 @@ def run(self, parsed_args):
         logger.info(
             "Revision %d of charm %r released to %s",
             revision, charm_name, ", ".join(parsed_args.channels))
+
+
+class StatusCommand(BaseCommand):
+    """List released revisions for a charm."""
+    name = 'status'
+    help_msg = "list released revisions for a charm in the Store"
```

Comment:
I think
```suggestion
    help_msg = "list released revisions of a charm"
```
would be better (hadn't we talked about how "in the Store" doesn't add anything?)

Reply:
Improved! (no, IIRC we talked about [sS]tore).